blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Customization of Blazor Rich Text Editor Dialog | Syncfusion
+description: Check out and learn here all about DialogSettings in Syncfusion Blazor RichTextEditor component and more.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# Customization of Blazor Rich Text Editor Dialog
+
+The Blazor Rich Text Editor allows for improved customization of the dialogs by configuring the RichTextEditorDialogSettings property. This addition enables users to control the behavior and appearance of the underlying dialogs within the RichTextEditor, the following list of options has been provided.
```

Comment:
The Blazor Rich Text Editor allows for improved customization of the dialogs by configuring the RichTextEditorDialogSettings property. This addition enables users to control the behavior and appearance of the underlying dialogs within the RichTextEditor. The following list of options has been provided.